deb/install/prerm

```diff
@@ -5,3 +5,7 @@ mv /etc/exports.old /etc/exports
 
 rm /etc/dnsmasq.conf
 mv /etc/dnsmasq.conf.old /etc/dnsmasq.conf
+
+systemctl stop dnsmasq
```

Comment:
in a `postrm` do we want to start these services back up? (in-case someone else is using them?). or you can do a `restart` right here?  or would a `restart` in `prerm` be bad cause your Debian package fils have not been removed yet?

Reply:
Lets add the `|| true` to the prerm optional items (like the restarts)